content/en/profiler/guide/isolate-outliers-in-monolithic-services.md

```diff
@@ -74,20 +74,50 @@ The APM `Trace operation` attribute lets you filter and group a flame graph with
 
 In the previous image, notice that the `ModelTraining` operation is taking more CPU time than its primary use in the `GET /train` endpoint, so it must be used elsewhere. Click the operation name to determine where else it is used. In this case, `ModelTraining` is also use by `POST /update_model`.
 
-## Isolate your own business logic (Java)
+
+## Isolate your own business logic
+
+{{< programming-lang-wrapper langs="java,go" >}}
+{{< programming-lang lang="java">}}
 
 Endpoint and operation isolation is available in your profiles by default, but you may want to isolate a different piece of logic. For example, if the monolith is sensitive to specific customers, you can add a custom filter to the profiles:
```

Comment:
I think we can keep this introduction above the tabs. It is relevant to both languages